mobile/test/cc/unit/fetch_client_test.cc

```diff
@@ -0,0 +1,21 @@
+#include <string>
+#include <vector>
+
+#include "examples/cc/fetch_client/fetch_client.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Platform {
+namespace {
+
+// This test verifies that the fetch client is able to successfully
+// build and start the Envoy engine. It will panic if it is unable
+// to do so.
+TEST(FetchClientTest, Foo) {
+  Envoy::Fetch client;
+  client.fetch({"https://wwwfff.google.com/"});
```

Comment:
just curious, is there a reason for the "fff"?

Reply:
*facepalm* It was supposed to be www, but I decided to test what would happen if the client could not reach the server (in this 'cause because DNS failed). And then I forgot to switch it back. Fixed. Thanks!